<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySimIIGui::SolverSettings</class>
 <widget class="QWidget" name="MySimIIGui::SolverSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Solver settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="TabWidget_1">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>500</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="TabContent1">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_Tab1">
       <item row="0" column="0">
        <widget class="QLabel" name="label_Space1">
         <property name="maximumSize">
          <size>
           <width>115</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string> </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Label_Solvers">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Solvers:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QComboBox" name="ComboBox_Solvers">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>EM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Heat</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EM+Heat</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CF+Heat</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EM+CF+Heat</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Label_Type">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QComboBox" name="ComboBox_Type">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Steady</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Transient</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Scanning</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <layout class="QHBoxLayout" name="HBox_WorkDirectory">
         <item>
          <widget class="QLabel" name="Label_WorkDirectory">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Work dir.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Gui::FileChooser" name="FileChooser_Dir" native="true">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>25</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="mode" >
            <enum>Gui::FileChooser::Directory</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_Space2">
         <property name="maximumSize">
          <size>
           <width>115</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string> </string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QRadioButton" name="IterativeSolver">
         <property name="maximumSize">
          <size>
           <width>115</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>IterativeSolver</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3" colspan="3">
        <widget class="QRadioButton" name="DirectiveSolver">
         <property name="maximumSize">
          <size>
           <width>115</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>DirectiveSolver</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QLabel" name="label_Space">
         <property name="text">
          <string>Computational steps (Sec./GHz) </string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="Label_Starting">
         <property name="text">
          <string>Starting：</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QLineEdit" name="LineEdit_Starting"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="Label_Ending">
         <property name="text">
          <string>Ending：</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QLineEdit" name="LineEdit_Ending"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="Label_Step">
         <property name="text">
          <string>Step：</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QLineEdit" name="LineEdit_Step"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="Label_OutputSteps">
         <property name="text">
          <string>OutputSteps：</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="2">
        <widget class="QLineEdit" name="LineEdit_OutputSteps"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="Label_MaxIterSteps">
         <property name="text">
          <string>MaxIterSteps：</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1" colspan="2">
        <widget class="QLineEdit" name="LineEdit_MaxIterSteps"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="Label_ConvergRelatErr">
         <property name="text">
          <string>ConvCr.: </string>
         </property>
        </widget>
       </item>
       <item row="12" column="1" colspan="2">
        <widget class="QLineEdit" name="LineEdit_ConvergRelatErr"/>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_Space3">
         <property name="maximumSize">
          <size>
           <width>115</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string> </string>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="3">
        <widget class="QLabel" name="label_ScalingSpace">
         <property name="text">
          <string>Coord. scaling (meter) </string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="Label_Scaling">
         <property name="text">
          <string>Scal.Coef.：</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1" colspan="2">
        <widget class="QLineEdit" name="LineEdit_Scaling">
         <property name="text">
          <string>0.01</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="TabContent2">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QGridLayout" name="GridLayout_Advanced">
       <item row="1" column="0">
        <widget class="QLabel" name="Label_Advanced">
         <property name="maximumSize">
          <size>
           <width>115</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Elmer SIF Prototype:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTextEdit" name="TextEdit_Advanced">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>300</height>
          </size>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="HBoxLayout_Ok">
     <item row="1" column="0">
      <spacer name="HorizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="PushButton_Ok">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="HorizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::FileChooser</class>
   <extends>QWidget</extends>
   <header>Gui/FileDialog.h</header>
  </customwidget>
 </customwidgets>
</ui>
